requirements.txt

```diff
@@ -2,4 +2,5 @@ click
 click-plugins
 colorama
 requests>=2.2.1
-XlsxWriter
\ No newline at end of file
+XlsxWriter
+ipaddress;python_version<='2.7'
```

Comment:
Example of this in action
py3 ignores the line for `ipaddress`

```
$ pip3 install -r requirements.txt 
Ignoring ipaddress: markers 'python_version <= "2.7"' don't match your environment
Requirement already satisfied: click in ..python3.7/site-pages (from -r requirements.txt (line 1)) (7.0)
Requirement already satisfied: click-plugins in ..python3.7/site-pages (from -r requirements.txt (line 2)) (1.1.1)
Requirement already satisfied: colorama in ..python3.7/site-pages (from -r requirements.txt (line 3)) (0.4.1)
Requirement already satisfied: requests>=2.2.1 in /usr/lib/python3/dist-packages (from -r requirements.txt (line 4)) (2.21.0)
Requirement already satisfied: XlsxWriter in ..python3.7/site-pages (from -r requirements.txt (line 5)) (1.2.1)
```

And for py27 it actually came along when I installed it on Ubuntu 19.04 as dist-packages, but it still evaluated it for installation.
```
$ pip2 install -r requirements.txt 
Requirement already satisfied: click in ..python2.7/site-pages (from -r requirements.txt (line 1)) (7.0)
Requirement already satisfied: click-plugins in ..python2.7/site-pages (from -r requirements.txt (line 2)) (1.1.1)
Requirement already satisfied: colorama in ..python2.7/site-pages (from -r requirements.txt (line 3)) (0.4.1)
Requirement already satisfied: requests>=2.2.1 in ..python2.7/site-pages (from -r requirements.txt (line 4)) (2.22.0)
Requirement already satisfied: XlsxWriter in ..python2.7/site-pages (from -r requirements.txt (line 5)) (1.2.1)
Requirement already satisfied: ipaddress in /usr/lib/python2.7/dist-packages (from -r requirements.txt (line 6)) (1.0.17)
Requirement already satisfied: idna<2.9,>=2.5 in /usr/lib/python2.7/dist-packages (from requests>=2.2.1->-r requirements.txt (line 4)) (2.6)
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in ..python2.7/site-pages (from requests>=2.2.1->-r requirements.txt (line 4)) (3.0.4)
Requirement already satisfied: certifi>=2017.4.17 in ..python2.7/site-pages (from requests>=2.2.1->-r requirements.txt (line 4)) (2019.9.11)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in ..python2.7/site-pages (from requests>=2.2.1->-r requirements.txt (line 4)) (1.25.6)
```